superset/connectors/sqla/models.py

```diff
@@ -360,10 +360,21 @@ def order_by_choices(self) -> list[tuple[str, str]]:
     def verbose_map(self) -> dict[str, str]:
         verb_map = {"__timestamp": "Time"}
         verb_map.update(
-            {o.metric_name: o.verbose_name or o.metric_name for o in self.metrics}
+            {
+                o.metric_name: verb_map.setdefault(
```

Comment:
not a big fan for `setdefault`, somehow it is super cryptic to me, had to read the docs and it's "get the value but set it if it doesn't exist". Ambiguous trying to be a getter and a setter at the same time. Can we go more verbose/procedural here using the normal getter/setter-type primitives?